src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -248,50 +251,93 @@ export class BaselinesTable extends Component {
         return table;
     }
 
+    renderEmptyState = (permissions) => {
+        const { baselineError, emptyState, loading, revertBaselineFetch } = this.props;
+        const { errorMessage } = this.state;
+
+        if (!baselineError.status) {
+            return <EmptyStateDisplay
+                icon={ AddCircleOIcon }
+                title={ EMPTY_BASELINES_TITLE }
+                text={ EMPTY_BASELINES_MESSAGE }
+                button={ <CreateBaselineButton
+                    emptyState={ emptyState }
+                    permissions={ permissions }
+                    loading={ loading } /> }
+            />;
+        } else if (baselineError.status !== 200 && baselineError.status !== undefined) {
```

Comment:
You have this same logic and error state here: https://github.com/RedHatInsights/drift-frontend/pull/573/files#diff-a6250f0578def843af6f5bfc368610706b808829808f1a607a19f1199c858f4cR96

You should only need it in `BaselinesTable`, so figure out how you can remove it from Baselines page altogether. At this point, `BaselinesPage` shouldn't be worried about an empty state at all.